cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,23 @@
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
```

Comment:
It will just try to read `install_manifest.txt` and remove all the files in the file. If the file doesn't exist, it will just prints error message "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt".